packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -53,8 +53,15 @@ export class AtomicResultList implements InitializableComponent {
   }
 
   private get defaultFieldsToInclude() {
-    // TODO: add more default fields to include
-    return ['parents'];
+    return [
+      'date',
+      'author',
+      'source',
+      'language',
+      'filetype',
+      'parents',
+      'ec_price',
+    ];
```

Comment:
What other field(s) could be default?

Reply:
The best reference we have is: https://docsstaging.coveo.com/COM-627-new-flow-of-catalog-creation/3139/coveo-for-commerce/creating-a-coveo-commerce-catalog#standard-fields-reference

It is a QA link and it should be merged this week or so

Reply:
Great thanks!